roles/pulp_health_check/tasks/main.yml

```diff
@@ -9,10 +9,22 @@
   debug:
     msg: '{{ ansible_facts.services|dict2items|selectattr("key", "contains", "pulpcore")|list|items2dict }}'
 
+- name: Services to exclude from verification
+  set_fact:
+    services_to_exclude:
+      - 'pulpcore.service'
+      - 'pulpcore-worker@.service'
+      - "{{ __pulp_version is version('3.16', '>=') | ternary('pulpcore-resource-manager.service', '') }}"
```

Comment:
Umm, we should delete the service instead. Once it is no longer needed. Thanks to `pulpcore.service`, we can do this.

Either in a separate PR, or in this one.